.pre-commit-config.yaml

```diff
@@ -24,6 +24,15 @@ repos:
     hooks:
       - id: isort
         name: isort
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: requirements-txt-fixer
+      - id: check-json
```

Comment:
What do these two hooks do?

Reply:
requirements-txt-fixer - sorts entries in requirements.txt.

check-json - checks json files for parseable syntax.